src/main/java/edu/oakland/production/middleware01/MiddlewareGisInterfaceImplementation.java

```diff
@@ -0,0 +1,45 @@
+package edu.oakland.production.middleware01;
+
+import edu.oakland.helper.admin.Satellite;
+import edu.oakland.production.middleware01.MiddlewareGisInterface;
+
+/**
+ * This class shall recieve that GPS signal from display and pass it to GIS processor.
+ *
+ */
+
```

Comment:
Generally you don't want any space between the Javadoc and what it describes, as some IDEs view it differently if there is a space there.
```suggestion
```